### PR TITLE
organ sprite touch-ups

### PR DESCRIPTION
**Pull request by muburu**
## About the PR
adds inhand sprites for a bunch of organs (made by me ofcourse). pretty much only includes organs obtainable by gibbing, weird organs that need to be mapped or spawned in like appendix or tongue aren't included (they are tiny anyways so idk how an inhand would look for them).

also adds unique sprites for species-specific organs that were missing them, diona lungs and vox lungs.
also also adds unique descriptions for organs that used default descriptions.

_this is my first pr so if i have done something blatantly wrong or stupid i apologize_

## Why / Balance
partially resolvves https://github.com/space-wizards/space-station-14/issues/31788
organs didn't have any inhands
## Technical details
just yml changes, png additions and new rsi for vox organs.
brains actually had inhand sprites before this pr but had no way of using them and the sprites were also bad so i replaced them

## Media
organs

- some white parts might be hard to see if you use github light
- technically different organs that don't have unique sprites (i.e: "animal stomach") use these inhands as well
- some organs with different organ sprites use regular inhands because the inhands would look basically the same
- slime organ inhands are also transparent
- feel free to yell at me if you think some of this is bad and i'll try to make it better
- grey men not included in pr
new lungs
new diona and vox lungs here

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

### CODE REVIEW

File: Resources/Prototypes/Body/Organs/Animal/animal.yml [hunk 1]
```diff
@@ -113,6 +122,9 @@
     groups:
     - id: Alcohol
       rateModifier: 0.1
+  - type: Item
+    size: Normal
```
Comment (lzk228) [inline Resources/Prototypes/Body/Organs/Animal/animal.yml:126]: i think would be better to leave normal for all organs

Comment (muburu) [reply Resources/Prototypes/Body/Organs/Animal/animal.yml]: makes sense

Comment (IProduceWidgets) [reply Resources/Prototypes/Body/Organs/Animal/animal.yml]: This does beg the question though, can you fit a spessman in a duffelbag?

**Review by UbaserB — CHANGES_REQUESTED** [review]
Can you make the brain and liver sprites a bit smaller?

**Comment by muburu** [comment]
should be good to merg if you think everything looks good

File: Resources/Prototypes/Body/Organs/diona.yml [hunk 2]
```diff
@@ -29,8 +29,11 @@
   id: OrganDionaBrain
   parent: [BaseDionaOrgan, OrganHumanBrain]
   name: brain
-  description: "The source of incredible, unending intelligence. Honk."
+  description: "The central hub of a diona's pseudo-neurological activity, its root-like tendrils search for its former body."
```
Comment (UbaserB) [inline Resources/Prototypes/Body/Organs/diona.yml:32]: it’s

**Review by UbaserB — APPROVED** [review]
sprite approval

File: Resources/Prototypes/Body/Organs/diona.yml (hunk 2 above)
Comment (muburu) [reply Resources/Prototypes/Body/Organs/diona.yml:32]: erm akshually the possessive form of it does not contain a '
image

**Comment by slarticodefast** [comment]
These look great, thank you for all the work and your first contribution!
I went ahead and removed the changelog entry from your PR description since sprite changes without gameplay impact should not get logged according to the PR guidelines.

**Pull request merged**